### DISCUSSION TITLE
Doppelganger (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#748] [Role Ideas]
- Der Doppelgänger startet im Spiel mit der CopyCatDeagle. Diese kopiert die Rolle des angeschossenen und der Doppelgänger spielt fortan mit seiner neuen Rolle weiter. (auch die Rolle des Detektiv)

- Der Doppelgänger hat Anfangs seine eigene Rolle, gehört also nicht Innocents, Traitoren bzw. anderen Fraktionen an,  wird in der Verteilung der Rollen als Innocent mitgezählt

- Im Falle des Jesters spielen die beiden unabhängig voneinander, es kann nur einen geben der gewinnt
- Wird der DG zum Sidekick geschossen, behält er die CopyCatDeagle und kann diese weiterhin benutzen
- Als Sidekick wird er auch zum Jackal, sobald dieser stirbt
- Als Infected (Host)(DG) , sterben alle infizierten, wenn er stirbt. Der andere Infected (Host) bleibt am leben. Und umgekehrt.

- Die Munition der CopyCatDeagle wird nur dann verbraucht, wenn er jemanden trifft. Er muss die Rolle eines anderen kopieren. Er kann nicht alleine mit der Rolle DG gewinnen. Ist er der letzte gewinnen die Innocents

- Wenn die neue angenommene Rolle einen Shop besitzt, erhält der Doppelgänger nur einen Credit 
(mit Anpassungsmöglichkeit)

- Es sollte eine anpassbare Wahrscheinlichkeit geben wann der DG erscheint, ab wie vielen Spielern und er ersetzt dann den Platz eines Innocents

Idea by: @Imperatoerchen

**Comment by TaintedEnergy** [comment]
A role with the name "doppelganger" has already been created, and either it or the mimic has similar abilities to the ones discussed here: https://steamcommunity.com/sharedfiles/filedetails/?id=2339261523
